src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -216,11 +218,16 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach with TableDrivenPro
       IngestParserTest().handleRequest(event(), null)
       val serveEvents = s3Server.getAllServeEvents.asScala
 
-      def filterEvents(name: String) = serveEvents
+      def getContentOfAllMetadataFilePutEvents: List[String] = serveEvents
         .map(_.getRequest)
-        .find(ev => ev.getUrl == s"/$testOutputBucket/$reference/$name" && ev.getMethod == RequestMethod.PUT)
+        .filter { ev =>
+          val outputBucket = s"/$testOutputBucket/$reference/"
+          // Currently assuming that all metadata files names contain a "." in them for the extension
```

Comment:
Do we need this comment? It sounds like we're going to have metadata without a "." in the future.

Reply:
Not really, I just had the thought that we would have a file without a "." in the future and then someone would be trying to figure out why the tests weren't passing but it's unlikely so I'll remove it